namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-preprod/resources/s3-metrics.tf

```diff
@@ -0,0 +1,98 @@
+/*
+ Based on https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/tree/main/example
+ */
+module "book_a_secure_move_metrics_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.5"
+
+  team_name              = var.team_name
+  business-unit          = "Digital and Technology"
```

Comment:
The value of `business-unit` must be one of the values listed in the guidance here:
https://ministryofjustice.github.io/technical-guidance/documentation/standards/documenting-infrastructure-owners.html#mandatory
In this case, I think `HMPPS` is probably the one you want.

Reply:
Now updated 👍